cli/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification-pipeline.yaml

```diff
@@ -94,5 +97,10 @@ jobs:
       # label_smoothing_factor: 0.0
       # max_grad_norm: 1.0
 
+      # Model evaluation args
```

Comment:
Why comment the model eval args?

Reply:
default value is label and image_url in pipeline component: https://github.com/Azure/azureml-assets/pull/1154/files